FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,15 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				let result = RemoteFeedLoaderParser.map(data: data, response: response)
+				completion(result)
```

Comment:
The `result` variable is not needed, you can inline it.

```suggestion
				completion(RemoteFeedLoaderParser.map(data: data, response: response))
```